service/storage_service.hh

```diff
@@ -795,6 +795,11 @@ private:
     // coordinator fiber
     future<> raft_state_monitor_fiber(raft::server&, sharded<db::system_distributed_keyspace>& sys_dist_ks);
 
+public:
+    bool topology_global_queue_empty() {
+        return not _topology_state_machine._topology.global_request.has_value();
```

Comment:
nit: the use of the `not` keyword took me by surprise.
I don't think we have anything in our coding style saying it should be avoided but it's the first time I see it used in our code base.

@avikivity / @scylladb/scylla-maint what's the official policy about [alternative operator representations](https://en.cppreference.com/w/cpp/language/operator_alternative)?

Reply:
Maybe we can just use positive logic here rather than negative logic, so to expose a `has_topology_global_queue` predicate instead.

Reply:
> @avikivity / @scylladb/scylla-maint what's the official policy about [alternative operator representations](https://en.cppreference.com/w/cpp/language/operator_alternative)?

We don't use it.

Reply:
@denesb "We don't use it" as in "it's forbidden" or "nobody used it before, go ahead and be the first"?

@bhalevy I was following `std::`-containers' `empty()` convention.

Reply:
Replaced `not ` with `!`.